server/src/test/ZetaSqlWrapper.spec.ts

```diff
@@ -148,4 +153,18 @@ describe('ZetaSqlWrapper', () => {
       shouldRegisterInformationSchema(tableDefinition, undefined, tableName);
     }
   });
+
+  it('register should update columns', () => {
```

Comment:
Can you split this test to several to have only one arrange/act/assert part in each test?